### ISSUE TITLE
ShippingMethod Zones not set

### ISSUE THREAD

**Issue by sshibani** [labd/terraform-provider-commercetools#67]
The Zone set on the ShippingMethod is mandatory for setting ShippingMethodRates. 

```
resource "commercetools_shipping_zone_rate" "standard-de" {
  shipping_method_id = "${commercetools_shipping_method.standard.id}"
  shipping_zone_id   = "${commercetools_shipping_zone.de.id}"

  price {
    cent_amount   = 5000
    currency_code = "EUR"
  }
}
```

`shipping_zone_id` Should be set on the ShippingMethod before the Rate can be set. [commercetools action](https://docs.commercetools.com/http-api-projects-shippingMethods#add-zone)

When creating a new ShippingMethod the Zones Array is not set. I think it would make more sense to set the Zone when creating the Rate.

**Comment by andrei-selilo** [comment]
Having an issue with this part, on `commercetools_shipping_zone_rate` creation.
Error: `Shipping method does not contain zone '{id of the zone}'`
So i have to manually `addZone` to the `commercetools_shipping_method` and after that - it will work.
Can you provide the details of how to add zone to the shipping method via terraform?

Code example:
```
resource "commercetools_shipping_method" "standard" {
  tax_category_id = "${commercetools_tax_category.standard.id}"
  name = "Standard ${terraform.workspace}"
  key = "shipping-method"
  is_default = true
  description = "standard shipping method ${terraform.workspace}"
}

resource "commercetools_shipping_zone" "uk" {
  name = "Zone UK ${terraform.workspace}"
  key = "shipping-zone"
  description = "UK shipping zone ${terraform.workspace}"
  location = {
      country = "GB"
  }
}

resource "commercetools_shipping_zone_rate" "standard-uk" {
  shipping_method_id = "${commercetools_shipping_method.standard.id}"
  shipping_zone_id   = "${commercetools_shipping_zone.uk.id}"

  price = {
    currency_code = "GBP"
    cent_amount = 1000
  }

  free_above = {
    cent_amount   = 50000
    currency_code = "GBP"
  }
}
```

**Comment by davidweterings** [comment]
@andrei-selilo this should be fixed now with the latest release (0.11.1)! Please try again and let us know if it doesn't work.

**Comment by andrei-selilo** [comment]
@davidweterings it works, thanks 👌🏻👍🏻
